test/functionalTest/cases/2452_q_for_attr_with_colon_in_name/q_for_attr_with_colon_in_name.test

```diff
@@ -0,0 +1,108 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+q for attr with colon in name
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create entity E with attribute A:1
+# 02. GET /v2/entities?q=A:1, no hits (no attr named A)
+# 03. GET /v2/entities?q='A:1', see the entity E
```

Comment:
Looking to @LeonanCarvalho report in #2452 , it would be great to include a case to ensure it works with negation, i.e.

GET /v2/entities?q=!'A:2'

works